WMF Framework/CommonStrings.swift

```diff
@@ -237,8 +237,12 @@ public class CommonStrings: NSObject {
     public static let viewFullHistoryText = WMFLocalizedString("aaald-view-full-history-button", value: "View full article history", comment: "Text displayed in a button for pushing to the full article history view on the article as a living document screen.")
     static let smallChangeDescription = WMFLocalizedString(
         "aaald-small-change-description",
-        value:"{{PLURAL:%1$d|0=No small changes|%1$d small change|%1$d small changes}} made",
+        value:"{{PLURAL:%1$d|0=No small changes|%1$d small change|%1$d small changes}}",
         comment:"Describes how many small changes are batched together in the article as a living document timeline view. %1$d is replaced with the number of small changes.")
+    static let smallChangeDescriptionSuffix = WMFLocalizedString(
```

Comment:
Good catch, I tried to fix this as best I could while preserving the partial link design. Now there's an additional formatter string to allow them to reorder each part. I also tried to add some instructions in a translation comment for languages where this doesn't make sense, where this will fall back to the entire description being a link.

Reply:
This still feels a little off, as we're assuming "made", "5", and "small changes" can be rearranged to make what is needed in all languages, when if some language's construction is essentially "small made changes" we'd be unable to recreate it.

Could we combine these into a "X small changes made" string - so that there is only one translation, which gets a number dropped into it?

Reply:
Yeah I know, it is annoyingly complicated. The only thing I'm trying to retain is keeping "x small changes" with the blue link styling, and "made" with the gray text styling, like the [mocks](https://app.zeplin.io/project/57a120ce9787dcf26230651f/screen/5eab3bd10825071dcda6c2be). I'm not sure how to do that without splitting up the strings so I can seek out the linked text later for styling. I did add an extra-long translations comment to "aaald-small-change-description-format", which explains a workaround for other languages if they want to bypass this weirdness. Let me know if you think of of a simpler way to retain the split-styling on this string, I may be over-complicating it. Maybe we add a comment to rejigger this if the experiment succeeds and we expand it to other languages?

Reply:
What if we just had it all blue for now, so just one string to translate? Let's confirm w/ @carolynlimadeo to make sure that is acceptable, but that seems easiest given the complexity here.

Reply:
Simplified label work is done in [88cc00](https://github.com/wikimedia/wikipedia-ios/pull/3683/commits/88cc009c1ff8fcb3a8cd637634bbb06004016ef3).

Reply:
Looks great!